### PR TITLE
Add verification step for authentication docs

### PR DESCRIPTION
**Pull request by sheajamba**
## Description
Adds a verification step to the Trino security overview page with a link to the steps to verify user/password configuration.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/security/password-file.rst [hunk 1]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
 
     htpasswd -B -C 10 password.db test
 
+.. _verify_authentication:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify password file authentication, log in to the :doc:`Web UI
+</admin/web-interface>`, and connect to the Trino :doc:`CLI </client/cli>`.
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:99]: and connect with the Trino CLI to the cluster

File: docs/src/main/sphinx/security/password-file.rst [hunk 2]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
...
+  and password combination.
+
+* Connect with the Trino CLI using a URL that uses HTTPS, such as
+  ``https://trino.example.net:8443`` with the addition of the ``--user`` and
+  ``--password`` properties:
+
+.. code-block:: text
+
+    ./trino --server https://trino.example.com:8443 --user test --password
+    Password:
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:114]: remove that

File: docs/src/main/sphinx/security/password-file.rst [hunk 3]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
...
+
+Send a query:
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:116]: Once connected .. you can send query to test

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/security/password-file.rst [hunk 4]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
...
     htpasswd -B -C 10 password.db test
 
+.. _verify_authentication:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify password file authentication, log in to the :doc:`Web UI
+</admin/web-interface>`, and connect with the Trino :doc:`CLI </client/cli>` to
+the cluster.
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:100]:
```suggestion
the cluster:
```

File: docs/src/main/sphinx/security/password-file.rst [hunk 5]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
...
+
+* Connect to the Web UI from your browser using a URL that uses HTTPS, such as
+  ``https://trino.example.com:8443``. Enter a username in the ``Username`` text
+  box and the corresponding password in the ``Password`` text box, and log in to
+  the UI. Confirm that you are not able to log in using an incorrect username
+  and password combination.
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:106]: Add that the username is shown in the top right corner after logging in successfully.

File: docs/src/main/sphinx/security/password-file.rst [hunk 6]
```diff
@@ -90,3 +90,43 @@ Add or update the password for the user ``test``:
...
+
+* Connect with the Trino CLI using a URL that uses HTTPS, such as
+  ``https://trino.example.net:8443`` with the addition of the ``--user`` and
+  ``--password`` properties:
+
+.. code-block:: text
+
+    ./trino --server https://trino.example.com:8443 --user test --password
+
```
Comment (mosabua) [inline docs/src/main/sphinx/security/password-file.rst:115]: Add that this quotes you for the password and after supplying you get the trino> prompt from the CLI and you can proceed to run a query

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now as well

**Review by electrum — APPROVED** [review]

**Pull request merged**